plugins/org.jboss.tools.openshift.core/META-INF/MANIFEST.MF

```diff
@@ -5,13 +5,19 @@ Bundle-SymbolicName: org.jboss.tools.openshift.core
 Bundle-Version: 3.0.0.qualifier
 Bundle-Activator: org.jboss.tools.openshift.internal.core.OpenShiftCoreActivator
 Bundle-Vendor: %Bundle-Vendor
-Require-Bundle: org.jboss.tools.openshift.common.core;bundle-version="[3.0.0,4.0.0)",
- org.jboss.tools.openshift.client;bundle-version="[3.0.0,4.0.0)",
+Require-Bundle: org.jboss.tools.openshift.client;bundle-version="[3.0.0,4.0.0)",
  org.jboss.tools.common.core;bundle-version="3.6.0",
- org.eclipse.core.runtime
+ org.eclipse.core.runtime,
+ org.jboss.tools.openshift.common.core;bundle-version="3.0.0",
+ org.apache.httpcomponents.httpclient;bundle-version="4.3.6",
+ org.apache.httpcomponents.httpcore;bundle-version="4.3.3",
+ org.apache.commons.codec;bundle-version="1.6.0",
+ org.jboss.ide.eclipse.as.dmr;bundle-version="3.1.0",
```

Comment:
@adietish agree...don't we have the impediment that we need to be concerned with people who have existing v2 connections.  I prefer the json approach I took because its more explicit, but moving v2 to that format would require some sort of migration right? In regards to DMR dependency, I didnt see an issue since we already implicitly depend on it in OSJC